src/vis/vis-view.js

```diff
@@ -223,6 +223,14 @@ var Vis = View.extend({
       return overlay.order === null ? Number.MAX_VALUE : overlay.order;
     });
 
+    // if logo is not included, let add it :)
+    var overlayNames = _.pluck(overlays, 'type');
+    if (!_.contains(overlayNames, 'logo')) {
```

Comment:
Take a look at [this part of the code where we're already adding new overlays to the vizjson object](https://github.com/CartoDB/cartodb.js/blob/v4/src/api/create-vis.js#L86). Perhaps worth adding this overlay there too? (just to have it in the same place)